fe/fe-core/src/main/java/com/starrocks/analysis/MultiRangePartitionDesc.java

```diff
@@ -136,7 +146,11 @@ public void setTimeUnit(String timeUnit) {
                     beginTime = beginTime.plusDays(timeInterval);
                     break;
                 case WEEK:
-                    partitionName = DEFAULT_PREFIX + beginTime.toString(DateUtils.WEEK_FORMAT);
+                    LocalDate localDate = LocalDate.of(beginTime.getYear(), beginTime.getMonthOfYear(),
+                            beginTime.getDayOfMonth());
+                    int weekOfWeekyear = localDate.get(weekFields.weekOfYear());
+                    partitionName = DEFAULT_PREFIX + String.format("%s_%02d",
+                            beginTime.toString(DateUtils.YEAR_FORMAT), weekOfWeekyear);
```

Comment:
DEFAULT_PREFIX can be concat in String.format too.
That can reduce an string concat operation.